src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,13 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+
+        Lottery loteria = new Lottery();
```

Comment:
Don't create several instances of the same class if you can use only one instance for your purposes. Here you can create a single instance of `Lottery` before the loop and use it within the loop.